### PR TITLE
Feat/url and content fixes

### PR DESCRIPTION
**Pull request by jakezu**
## Description :sparkles:
Fix broken links in public UI and update locale json files

### Related :handshake:
VEN-1579, VEN-1580

### CODE REVIEW

**Review by nikomakela — COMMENTED** [review]
The commits are not in conventional commit format, so they won't be recognised by release-please process. If there is one in Berth-reservation?

File: src/features/winterStorage/winterStoragePage/WinterStorageNotice.tsx [hunk 1]
```diff
@@ -33,27 +22,14 @@ const WinterStorageNotice = () => {
       <Row>
         <Col lg={{ size: 10, offset: 1 }} xl={{ size: 8, offset: 2 }}>
           <h3>{t('page.winter_storage.notice.title')}</h3>
+          <p>{t('page.winter_storage.notice.paragraph1')}</p>
+          <p>{t('page.winter_storage.notice.paragraph2')}</p>
```
Comment (nikomakela) [inline src/features/winterStorage/winterStoragePage/WinterStorageNotice.tsx:26]: This is not a good practise, because in different langauges there can be a different amount of paragraphs. The line breaks should be set in the translations instead and probably a `<pre>`-tag is needed then?

If this is something that works now and will never ever be changed, then this could be acceptable, but instead of using paragraph1 and paragraph2, something like "body text" or something that adds some context should be used instead.

File: src/features/winterStorage/winterStoragePage/WinterStorageNotice.tsx [hunk 2]
```diff
@@ -33,27 +22,14 @@ const WinterStorageNotice = () => {
...
-                >
-                  content
-                </a>,
-              ]}
-            />
-          </p>
-          <p>
-            <Trans
-              i18nKey={'page.winter_storage.notice.paragraph2'}
+              i18nKey={'page.winter_storage.notice.paragraph3'}
```
Comment (nikomakela) [inline src/features/winterStorage/winterStoragePage/WinterStorageNotice.tsx:29]: It is superhard to follow what the content actually is. I think there could be some hints about the context at least. The Trans tag has for example a [default attribute](https://www.hel.fi/en/culture-and-leisure/outdoor-activities-parks-and-nature-destinations/boating/winter-storage-of-boats).

**Review by nikomakela — APPROVED** [review]
I tested most of the links and they worked. I would change the translation keys and least give some context to them. Using paragraph1, paragraph2 and paragraph3 tells nothing about the content and there can be different amount of paragraphs when using other languages. Better translation keys and default-attributes would give some context. But, knowing the current status of Berth reservation, I think this is approvable.

**Comment by terovirtanen** [comment]
**Venepaikat-customer-ui branch is deployed to platta: https://venepaikat-customer-ui-pr510.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by jakezu** [comment]
We discussed with Niko and Jiri and concluded that the changes are acceptable as they are.
